flipperdevices/flipperzero-firmware
# [FL-1930] USB HID

# What's new

- USB HID keyboard + mouse descriptor
- API to send keyboard/mouse reports
- USB Mouse demo app
- RubberDucky script parser and BadUSB test app

# Verification 

- Connect USB, run USB mouse demo app, press keys and make sure the cursor is moving
- Name your Ducky script "badusb.txt" and copy it to sd card root
- Run Bad USB test app and make sure the script is executed correctly

# Checklist (do not modify)
